src/test/test.py

```diff
@@ -387,7 +387,7 @@ def test_genomic_biomarkers_building(self):
             xc.ASSAY_TYPE: "WGS",
             xc.COVERAGE: 80,
             xc.FAILED: False,
-            xc.ONCOKB_CACHE: oncokb_cache_params,
+            xc.ONCOKB_CACHE: oncokb_cache_params(),
```

Comment:
We want to input an `oncokb_cache_params` object, but instead the input was the `oncokb_cache_params` class -- which is _very_different and results in some rather cryptic Python errors